db/migrate/20230613121629_create_greetings.rb

```diff
@@ -0,0 +1,8 @@
+class CreateGreetings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :greetings do |t|
+
```

Comment:
- [ ] Your Greetings migration is not completed, I think you want to put `:message` as a string. Kinldy accepty my suggestion. 
```suggestion
      t.string :message
```